src/main/java/org/openstreetmap/atlas/tags/FerryTag.java

```diff
@@ -0,0 +1,47 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.EnumSet;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM Ferry Tag
+ *
+ * @author isabellehillberg
+ */
+@Tag(taginfo = "https://taginfo.openstreetmap.org/keys/ferry#values")
+public enum FerryTag
+{
+    YES,
+    NO,
+    MOTORWAY,
+    TRUNK,
+    PRIMARY,
+    SECONDARY,
+    TERTIARY,
+    UNCLASSIFIED,
+    RESIDENTIAL,
+    SERVICE,
+    FOOTWAY,
+    PEDESTRIAN,
+    TRACK;
+
+    private static final EnumSet<FerryTag> CAV_NAVIGABLE = EnumSet.of(MOTORWAY, TRUNK, PRIMARY,
+            SECONDARY, TERTIARY, UNCLASSIFIED, RESIDENTIAL, SERVICE);
+    private static final EnumSet<FerryTag> PEDESTRIAN_NAVIGABLE = EnumSet.of(FOOTWAY, TRACK,
+            PEDESTRIAN);
+
+    @TagKey
+    public static final String KEY = "ferry";
+
+    public static boolean isCarNavigableHighway(final FerryTag tag)
```

Comment:
Nit: What do you think of `isCarNavigableFerry`?

Reply:
Yes! copy fail from the highway tag :)